Update enable-attack-surface-reduction.md

cunninghamp opened the pull request:
Added clarification for where to find the OMA-URI value to use for the example ASR rule.

This was not immediately clear that the reader needed to reference another part of the page for the specific value to use when following the example.

cunninghamp committed:
Update enable-attack-surface-reduction.md

Added clarification for where to find the OMA-URI value to use for the example ASR rule.

yogkumgit commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

JohanFreelancer9 commented on microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md:
```diff
@@ -135,7 +135,7 @@ You can use Microsoft Endpoint Manager (MEM) OMA-URI to configure custom ASR rul
    - In **Name**, type a name for the rule.
    - In **Description**, type a brief description.
-   - In **OMA-URI**, type or paste the specific OMA-URI link for the rule that you are adding.
+   - In **OMA-URI**, type or paste the specific OMA-URI link for the rule that you are adding. Refer to the [MDM section of this page](https://docs.microsoft.com/en-gb/microsoft-365/security/defender-endpoint/enable-attack-surface-reduction?view=o365-worldwide#mdm) for the OMA-URI to use for this example rule.
```
```suggestion
   - In **OMA-URI**, type or paste the specific OMA-URI link for the rule that you are adding. Refer to the MDM section earlier in this topic for the OMA-URI to use for this example rule.
```

jweston-1 replied:
```suggestion
  - In **OMA-URI**, type or paste the specific OMA-URI link for the rule that you are adding. Refer to the MEM section earlier in this topic for the OMA-URI to use for this example rule. For ASR rule GUIDS, see [Per rule descriptions](attack-surface-reduction-rules.md#per-rule-descriptions) in the topic: Attack surface reduction rules.
```

JohanFreelancer9 reviewed (changes requested):
Copy review - @cunninghamp, kindly commit the suggested change. Many thanks.

jweston-1 committed:
Update microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md

Minor revision - OMA-URI is in MEM, not MDM. Added, link to per-rule GUID info.

jweston-1 committed:
Merge branch 'public' into patch-1
